### PR TITLE
Topic/test fix

### PR DESCRIPTION
**Pull request by shamisp**
Bug fix for #508

### CODE REVIEW

**Comment by shamisp** [comment]
autobot build

**Comment by shamisp** [comment]
@yosefe please take a look

**Comment by shamisp** [comment]
BTW, this has nothing to do with Power. We have to start testing with a various configurations/

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1153/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/566/ for details (Mellanox internal link).

File: test/gtest/ucs/test_mpool.cc [hunk 1]
```diff
@@ -47,6 +47,7 @@ UCS_TEST_F(test_mpool, no_allocs) {
     ucs_mpool_cleanup(&mp, 1);
 }
 
+#if ENABLE_DEBUG_DATA
```
Comment (shamisp) [inline test/gtest/ucs/test_mpool.cc]: I will move it to another location

**Comment by shamisp** [comment]
@yosefe - please take a look.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/567/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1155/ for details.

**Comment by yosefe** [comment]
:+1:

**Comment by shamisp** [comment]
I guess this error is unrelated:
```
 [     INFO ] 0 214 3000x{0..214} [1450210163.409913] [jenkins01:22374:0]       cm_iface.c:147  UCX  ERROR SIDR request error, status: response timeout error
```

**Comment by yosefe** [comment]
nope

**Pull request merged**